<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Get GWAS Catalog variants — get_variants • gwasrapidd</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



  <link href="../extra.css" rel="stylesheet">
  

<meta property="og:title" content="Get GWAS Catalog variants — get_variants" />
<meta property="og:description" content="Retrieves variants via the NHGRI-EBI GWAS Catalog REST API. The REST
API is queried multiple times with the criteria passed as arguments (see
below). By default all variants that match the criteria supplied in the
arguments are retrieved: this corresponds to the default option
set_operation set to 'union'. If you rather have only the
variants that match simultaneously all criteria provided, then set
set_operation to 'intersection'." />
<meta property="og:image" content="https://rmagno.eu/gwasrapidd/logo.svg" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gwasrapidd</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.8</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/gwasrapidd.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/bmi_variants.html">Variants associated with Body Mass Index (BMI)</a>
    </li>
    <li>
      <a href="../articles/faq.html">Frequently asked questions</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/ramiromagno/gwasrapidd/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Get GWAS Catalog variants</h1>
    <small class="dont-index">Source: <a href='https://github.com/ramiromagno/gwasrapidd/blob/master/R/get_variants.R'><code>R/get_variants.R</code></a></small>
    <div class="hidden name"><code>get_variants.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Retrieves variants via the NHGRI-EBI GWAS Catalog REST API. The REST
API is queried multiple times with the criteria passed as arguments (see
below). By default all variants that match the criteria supplied in the
arguments are retrieved: this corresponds to the default option
<code>set_operation</code> set to <code>'union'</code>. If you rather have only the
variants that match simultaneously all criteria provided, then set
<code>set_operation</code> to <code>'intersection'</code>.</p>
    </div>

    <pre class="usage"><span class='fu'>get_variants</span>(
  <span class='kw'>study_id</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>association_id</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>variant_id</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>efo_id</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>pubmed_id</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>genomic_range</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>cytogenetic_band</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>gene_name</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>efo_trait</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>reported_trait</span> <span class='kw'>=</span> <span class='kw'>NULL</span>,
  <span class='kw'>set_operation</span> <span class='kw'>=</span> <span class='st'>"union"</span>,
  <span class='kw'>interactive</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>,
  <span class='kw'>std_chromosomes_only</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>,
  <span class='kw'>verbose</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>,
  <span class='kw'>warnings</span> <span class='kw'>=</span> <span class='fl'>TRUE</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>study_id</th>
      <td><p>A character vector of GWAS Catalog study accession
identifiers.</p></td>
    </tr>
    <tr>
      <th>association_id</th>
      <td><p>A character vector of GWAS Catalog association
identifiers.</p></td>
    </tr>
    <tr>
      <th>variant_id</th>
      <td><p>A character vector of GWAS Catalog variant identifiers.</p></td>
    </tr>
    <tr>
      <th>efo_id</th>
      <td><p>A character vector of <a href='https://www.ebi.ac.uk/efo/'>EFO</a>
identifiers.</p></td>
    </tr>
    <tr>
      <th>pubmed_id</th>
      <td><p>An integer vector of
<a href='https://www.ncbi.nlm.nih.gov/pubmed'>PubMed</a> identifiers.</p></td>
    </tr>
    <tr>
      <th>genomic_range</th>
      <td><p>A named list of three vectors:</p><dl'>
<dt>chromosome</dt><dd><p>A character vector of chromosome names of the
form 1--22, X or Y.</p></dd> <dt>start</dt><dd><p>A numeric vector of start positions,
starting at 1.</p></dd> <dt>end</dt><dd><p>A numeric vector of end positions.</p></dd> 
</dl><p>The three vectors need to be of the same length so that <code>chromosome</code>
names, <code>start</code> and <code>end</code> positions can be matched by position.</p></td>
    </tr>
    <tr>
      <th>cytogenetic_band</th>
      <td><p>A character vector of cytogenetic bands of the form
<code>'1p36.11'</code>.</p></td>
    </tr>
    <tr>
      <th>gene_name</th>
      <td><p>Gene symbol according to
<a href='https://www.genenames.org/'>HUGO Gene Nomenclature (HGNC)</a>.</p></td>
    </tr>
    <tr>
      <th>efo_trait</th>
      <td><p>A character vector of
<a href='https://www.ebi.ac.uk/efo/'>EFO</a> trait descriptions, e.g.,
<code>'uric acid measurement'</code>.</p></td>
    </tr>
    <tr>
      <th>reported_trait</th>
      <td><p>A character vector of phenotypic traits as
reported by the original authors of the study.</p></td>
    </tr>
    <tr>
      <th>set_operation</th>
      <td><p>Either <code>'union'</code> or <code>'intersection'</code>. This
tells how variants retrieved by different criteria  should be combined:
<code>'union'</code> binds together all results removing duplicates and
<code>'intersection'</code> only keeps same variants found with different
criteria.</p></td>
    </tr>
    <tr>
      <th>interactive</th>
      <td><p>A logical. If all variants are requested, whether to ask
interactively if we really want to proceed.</p></td>
    </tr>
    <tr>
      <th>std_chromosomes_only</th>
      <td><p>Whether to return only variants mapped to
standard chromosomes: 1 thru 22, X, Y, and MT.</p></td>
    </tr>
    <tr>
      <th>verbose</th>
      <td><p>Whether the function should be
verbose about the different queries or not.</p></td>
    </tr>
    <tr>
      <th>warnings</th>
      <td><p>Whether to print warnings.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A <a href='variants-class.html'>variants</a> object.</p>
    <h2 class="hasAnchor" id="details"><a class="anchor" href="#details"></a>Details</h2>

    <p>Please note that all search criteria are vectorised, thus allowing for batch
mode search, e.g., one can search by multiple variant identifiers at once by
passing a vector of identifiers to <code>variant_id</code>.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='co'># Get variants by study identifier</span>
<span class='fu'>get_variants</span>(<span class='kw'>study_id</span> <span class='kw'>=</span> <span class='st'>'GCST001085'</span>)</div><div class='output co'>#&gt; An object of class "variants"
#&gt; Slot "variants":
#&gt; # A tibble: 18 x 7
#&gt;    variant_id merged functional_class chromosome_name chromosome_posi…
#&gt;    &lt;chr&gt;       &lt;int&gt; &lt;chr&gt;            &lt;chr&gt;                      &lt;int&gt;
#&gt;  1 rs10496288      0 intergenic_vari… 2                       83065441
#&gt;  2 rs7827545       0 intron_variant   8                      134554324
#&gt;  3 rs10785581      0 intron_variant   12                      45537190
#&gt;  4 rs200759        0 intron_variant   20                      15625776
#&gt;  5 rs7960483       0 intron_variant   12                      45531972
#&gt;  6 rs1372662       0 intron_variant   8                      134554803
#&gt;  7 rs10496289      0 intergenic_vari… 2                       83066256
#&gt;  8 rs6452524       0 intron_variant   5                       83137962
#&gt;  9 rs200752        0 intron_variant   20                      15618886
#&gt; 10 rs3798440       0 intron_variant   6                       75846902
#&gt; 11 rs7735940       0 regulatory_regi… 5                       36423829
#&gt; 12 rs9350602       0 intron_variant   6                       75850781
#&gt; 13 rs10188442      0 intron_variant   2                      132431666
#&gt; 14 rs12522034      0 intergenic_vari… 5                       36425491
#&gt; 15 rs13420028      0 intron_variant   2                      132430533
#&gt; 16 rs2469997       0 intergenic_vari… 8                      119341027
#&gt; 17 rs6887846       0 intron_variant   5                       83140542
#&gt; 18 rs6469823       1 intergenic_vari… 8                      119341744
#&gt; # … with 2 more variables: chromosome_region &lt;chr&gt;, last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "genomic_contexts":
#&gt; # A tibble: 148 x 12
#&gt;    variant_id gene_name chromosome_name chromosome_posi… distance is_mapped_gene
#&gt;    &lt;chr&gt;      &lt;chr&gt;     &lt;chr&gt;                      &lt;int&gt;    &lt;int&gt; &lt;lgl&gt;         
#&gt;  1 rs10496288 AC138623… 2                       83065441   153449 TRUE          
#&gt;  2 rs10496288 LOC10537… 2                       83065441   127905 FALSE         
#&gt;  3 rs10496288 AC098817… 2                       83065441   198933 TRUE          
#&gt;  4 rs10496288 LOC11226… 2                       83065441   148019 FALSE         
#&gt;  5 rs7827545  AC015599… 8                      134554324    43747 FALSE         
#&gt;  6 rs7827545  AC015599… 8                      134554324    43994 FALSE         
#&gt;  7 rs7827545  LOC10041… 8                      134554324   606901 FALSE         
#&gt;  8 rs7827545  AC105180… 8                      134554324   163837 FALSE         
#&gt;  9 rs7827545  ZFAT      8                      134554324        0 TRUE          
#&gt; 10 rs7827545  ZFAT-AS1  8                      134554324    43747 FALSE         
#&gt; # … with 138 more rows, and 6 more variables: is_closest_gene &lt;lgl&gt;,
#&gt; #   is_intergenic &lt;lgl&gt;, is_upstream &lt;lgl&gt;, is_downstream &lt;lgl&gt;, source &lt;chr&gt;,
#&gt; #   mapping_method &lt;chr&gt;
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 86 x 3
#&gt;    variant_id gene_name  ensembl_id     
#&gt;    &lt;chr&gt;      &lt;chr&gt;      &lt;chr&gt;          
#&gt;  1 rs10496288 AC138623.1 ENSG00000223977
#&gt;  2 rs10496288 AC098817.1 ENSG00000286211
#&gt;  3 rs7827545  AC015599.1 ENSG00000276140
#&gt;  4 rs7827545  AC015599.2 ENSG00000277732
#&gt;  5 rs7827545  AC105180.2 ENSG00000288067
#&gt;  6 rs7827545  ZFAT       ENSG00000066827
#&gt;  7 rs7827545  ZFAT-AS1   ENSG00000248492
#&gt;  8 rs7827545  AC015599.3 ENSG00000278454
#&gt;  9 rs10785581 AC079950.2 ENSG00000283545
#&gt; 10 rs10785581 ANO6       ENSG00000177119
#&gt; # … with 76 more rows
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 65 x 3
#&gt;    variant_id gene_name    entrez_id
#&gt;    &lt;chr&gt;      &lt;chr&gt;        &lt;chr&gt;    
#&gt;  1 rs10496288 LOC105374832 105374832
#&gt;  2 rs10496288 LOC112268410 112268410
#&gt;  3 rs7827545  LOC100419617 100419617
#&gt;  4 rs7827545  ZFAT         57623    
#&gt;  5 rs7827545  ZFAT-AS1     594840   
#&gt;  6 rs10785581 LOC105369744 105369744
#&gt;  7 rs10785581 ANO6         196527   
#&gt;  8 rs10785581 LOC100128248 100128248
#&gt;  9 rs10785581 LOC105369743 105369743
#&gt; 10 rs200759   MACROD2      140733   
#&gt; # … with 55 more rows
#&gt; </div><div class='input'>
<span class='co'># Get variants by association identifier</span>
<span class='fu'>get_variants</span>(<span class='kw'>association_id</span> <span class='kw'>=</span> <span class='st'>'25389945'</span>)</div><div class='output co'>#&gt; An object of class "variants"
#&gt; Slot "variants":
#&gt; # A tibble: 27 x 7
#&gt;    variant_id merged functional_class chromosome_name chromosome_posi…
#&gt;    &lt;chr&gt;       &lt;int&gt; &lt;chr&gt;            &lt;chr&gt;                      &lt;int&gt;
#&gt;  1 rs9486815       0 intron_variant   6                      108126842
#&gt;  2 rs4245535       0 intron_variant   6                      108128008
#&gt;  3 rs17069173      0 intergenic_vari… 6                      108017063
#&gt;  4 rs9374021       0 intron_variant   6                      108106220
#&gt;  5 rs9374007       0 intergenic_vari… 6                      108037124
#&gt;  6 rs9386694       0 intergenic_vari… 6                      108025473
#&gt;  7 rs9374002       0 non_coding_tran… 6                      108030386
#&gt;  8 rs218289        0 intron_variant   6                      108133233
#&gt;  9 rs1064346       0 3_prime_UTR_var… 6                      108043747
#&gt; 10 rs9374013       0 intron_variant   6                      108060143
#&gt; # … with 17 more rows, and 2 more variables: chromosome_region &lt;chr&gt;,
#&gt; #   last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "genomic_contexts":
#&gt; # A tibble: 388 x 12
#&gt;    variant_id gene_name chromosome_name chromosome_posi… distance is_mapped_gene
#&gt;    &lt;chr&gt;      &lt;chr&gt;     &lt;chr&gt;                      &lt;int&gt;    &lt;int&gt; &lt;lgl&gt;         
#&gt;  1 rs9486815  NR2E1     6                      108126842    39180 FALSE         
#&gt;  2 rs9486815  OSTM1     6                      108126842        0 TRUE          
#&gt;  3 rs9486815  OSTM1-AS1 6                      108126842        0 FALSE         
#&gt;  4 rs9486815  Z98200.2  6                      108126842    96124 FALSE         
#&gt;  5 rs9486815  SNX3      6                      108126842    84380 FALSE         
#&gt;  6 rs9486815  OSTM1-AS1 6                      108126842        0 TRUE          
#&gt;  7 rs9486815  NR2E1     6                      108126842    39180 FALSE         
#&gt;  8 rs9486815  OSTM1     6                      108126842    52101 FALSE         
#&gt;  9 rs9486815  SNX3      6                      108126842    84380 FALSE         
#&gt; 10 rs9486815  AL078596… 6                      108126842    52029 FALSE         
#&gt; # … with 378 more rows, and 6 more variables: is_closest_gene &lt;lgl&gt;,
#&gt; #   is_intergenic &lt;lgl&gt;, is_upstream &lt;lgl&gt;, is_downstream &lt;lgl&gt;, source &lt;chr&gt;,
#&gt; #   mapping_method &lt;chr&gt;
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 234 x 3
#&gt;    variant_id gene_name  ensembl_id     
#&gt;    &lt;chr&gt;      &lt;chr&gt;      &lt;chr&gt;          
#&gt;  1 rs9486815  NR2E1      ENSG00000112333
#&gt;  2 rs9486815  OSTM1      ENSG00000081087
#&gt;  3 rs9486815  OSTM1-AS1  ENSG00000225174
#&gt;  4 rs9486815  Z98200.2   ENSG00000271734
#&gt;  5 rs9486815  SNX3       ENSG00000112335
#&gt;  6 rs9486815  AL078596.1 ENSG00000279398
#&gt;  7 rs9486815  Z98200.1   ENSG00000238490
#&gt;  8 rs4245535  OSTM1-AS1  ENSG00000225174
#&gt;  9 rs4245535  AL078596.1 ENSG00000279398
#&gt; 10 rs4245535  NR2E1      ENSG00000112333
#&gt; # … with 224 more rows
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 154 x 3
#&gt;    variant_id gene_name    entrez_id
#&gt;    &lt;chr&gt;      &lt;chr&gt;        &lt;chr&gt;    
#&gt;  1 rs9486815  NR2E1        7101     
#&gt;  2 rs9486815  OSTM1        28962    
#&gt;  3 rs9486815  OSTM1-AS1    100287366
#&gt;  4 rs9486815  SNX3         8724     
#&gt;  5 rs9486815  LOC105377929 105377929
#&gt;  6 rs4245535  OSTM1-AS1    100287366
#&gt;  7 rs4245535  LOC105377929 105377929
#&gt;  8 rs4245535  NR2E1        7101     
#&gt;  9 rs4245535  SNX3         8724     
#&gt; 10 rs4245535  OSTM1        28962    
#&gt; # … with 144 more rows
#&gt; </div><div class='input'>
<span class='co'># Get a variant by its identifier</span>
<span class='fu'>get_variants</span>(<span class='kw'>variant_id</span> <span class='kw'>=</span> <span class='st'>'rs3798440'</span>)</div><div class='output co'>#&gt; An object of class "variants"
#&gt; Slot "variants":
#&gt; # A tibble: 1 x 7
#&gt;   variant_id merged functional_class chromosome_name chromosome_posi…
#&gt;   &lt;chr&gt;       &lt;int&gt; &lt;chr&gt;            &lt;chr&gt;                      &lt;int&gt;
#&gt; 1 rs3798440       0 intron_variant   6                       75846902
#&gt; # … with 2 more variables: chromosome_region &lt;chr&gt;, last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "genomic_contexts":
#&gt; # A tibble: 9 x 12
#&gt;   variant_id gene_name chromosome_name chromosome_posi… distance is_mapped_gene
#&gt;   &lt;chr&gt;      &lt;chr&gt;     &lt;chr&gt;                      &lt;int&gt;    &lt;int&gt; &lt;lgl&gt;         
#&gt; 1 rs3798440  RNA5SP209 6                       75846902    18337 FALSE         
#&gt; 2 rs3798440  RNU6-155P 6                       75846902    78744 FALSE         
#&gt; 3 rs3798440  AL392166… 6                       75846902    81611 FALSE         
#&gt; 4 rs3798440  IMPG1     6                       75846902    74212 FALSE         
#&gt; 5 rs3798440  RNU6-155P 6                       75846902    78744 FALSE         
#&gt; 6 rs3798440  RNA5SP209 6                       75846902    18337 FALSE         
#&gt; 7 rs3798440  MYO6      6                       75846902        0 FALSE         
#&gt; 8 rs3798440  IMPG1     6                       75846902    74212 FALSE         
#&gt; 9 rs3798440  MYO6      6                       75846902        0 TRUE          
#&gt; # … with 6 more variables: is_closest_gene &lt;lgl&gt;, is_intergenic &lt;lgl&gt;,
#&gt; #   is_upstream &lt;lgl&gt;, is_downstream &lt;lgl&gt;, source &lt;chr&gt;, mapping_method &lt;chr&gt;
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 5 x 3
#&gt;   variant_id gene_name  ensembl_id     
#&gt;   &lt;chr&gt;      &lt;chr&gt;      &lt;chr&gt;          
#&gt; 1 rs3798440  RNA5SP209  ENSG00000223169
#&gt; 2 rs3798440  RNU6-155P  ENSG00000252156
#&gt; 3 rs3798440  AL392166.1 ENSG00000200040
#&gt; 4 rs3798440  IMPG1      ENSG00000112706
#&gt; 5 rs3798440  MYO6       ENSG00000196586
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 4 x 3
#&gt;   variant_id gene_name entrez_id
#&gt;   &lt;chr&gt;      &lt;chr&gt;     &lt;chr&gt;    
#&gt; 1 rs3798440  RNA5SP209 100873469
#&gt; 2 rs3798440  RNU6-155P 106481224
#&gt; 3 rs3798440  IMPG1     3617     
#&gt; 4 rs3798440  MYO6      4646     
#&gt; </div><div class='input'>
<span class='co'># Get variants by EFO trait identifier</span>
<span class='fu'>get_variants</span>(<span class='kw'>efo_id</span> <span class='kw'>=</span> <span class='st'>'EFO_0005537'</span>)</div><div class='output co'>#&gt; An object of class "variants"
#&gt; Slot "variants":
#&gt; # A tibble: 8 x 7
#&gt;   variant_id merged functional_class chromosome_name chromosome_posi…
#&gt;   &lt;chr&gt;       &lt;int&gt; &lt;chr&gt;            &lt;chr&gt;                      &lt;int&gt;
#&gt; 1 rs2464195       0 missense_variant 12                     120997672
#&gt; 2 rs17215231      0 5_prime_UTR_var… 6                       33272092
#&gt; 3 rs10069690      0 intron_variant   5                        1279675
#&gt; 4 rs2363956       0 missense_variant 19                      17283315
#&gt; 5 rs4245739       0 3_prime_UTR_var… 1                      204549714
#&gt; 6 rs10771399      0 intergenic_vari… 12                      28002147
#&gt; 7 rs3757318       0 intron_variant   6                      151592978
#&gt; 8 rs78378222      0 3_prime_UTR_var… 17                       7668434
#&gt; # … with 2 more variables: chromosome_region &lt;chr&gt;, last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "genomic_contexts":
#&gt; # A tibble: 384 x 12
#&gt;    variant_id gene_name chromosome_name chromosome_posi… distance is_mapped_gene
#&gt;    &lt;chr&gt;      &lt;chr&gt;     &lt;chr&gt;                      &lt;int&gt;    &lt;int&gt; &lt;lgl&gt;         
#&gt;  1 rs2464195  HNF1A-AS1 12                     120997672    25380 FALSE         
#&gt;  2 rs2464195  CLIC1P1   12                     120997672    82349 FALSE         
#&gt;  3 rs2464195  SPPL3     12                     120997672    93314 FALSE         
#&gt;  4 rs2464195  AC069214… 12                     120997672    89735 FALSE         
#&gt;  5 rs2464195  LOC10537… 12                     120997672    71187 FALSE         
#&gt;  6 rs2464195  OASL      12                     120997672    21439 FALSE         
#&gt;  7 rs2464195  OASL2P    12                     120997672    56060 FALSE         
#&gt;  8 rs2464195  HNF1A-AS1 12                     120997672    16707 FALSE         
#&gt;  9 rs2464195  XLOC_009… 12                     120997672    89598 FALSE         
#&gt; 10 rs2464195  SPPL3     12                     120997672    93314 FALSE         
#&gt; # … with 374 more rows, and 6 more variables: is_closest_gene &lt;lgl&gt;,
#&gt; #   is_intergenic &lt;lgl&gt;, is_upstream &lt;lgl&gt;, is_downstream &lt;lgl&gt;, source &lt;chr&gt;,
#&gt; #   mapping_method &lt;chr&gt;
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 188 x 3
#&gt;    variant_id gene_name  ensembl_id     
#&gt;    &lt;chr&gt;      &lt;chr&gt;      &lt;chr&gt;          
#&gt;  1 rs2464195  HNF1A-AS1  ENSG00000241388
#&gt;  2 rs2464195  CLIC1P1    ENSG00000231313
#&gt;  3 rs2464195  SPPL3      ENSG00000157837
#&gt;  4 rs2464195  AC069214.1 ENSG00000286493
#&gt;  5 rs2464195  OASL       ENSG00000135114
#&gt;  6 rs2464195  OASL2P     ENSG00000283542
#&gt;  7 rs2464195  HNF1A      ENSG00000135100
#&gt;  8 rs2464195  AC079602.4 ENSG00000279001
#&gt;  9 rs2464195  AC079602.3 ENSG00000256963
#&gt; 10 rs2464195  C12orf43   ENSG00000157895
#&gt; # … with 178 more rows
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 91 x 3
#&gt;    variant_id gene_name    entrez_id
#&gt;    &lt;chr&gt;      &lt;chr&gt;        &lt;chr&gt;    
#&gt;  1 rs2464195  HNF1A-AS1    283460   
#&gt;  2 rs2464195  CLIC1P1      390363   
#&gt;  3 rs2464195  SPPL3        121665   
#&gt;  4 rs2464195  LOC105378258 105378258
#&gt;  5 rs2464195  OASL         8638     
#&gt;  6 rs2464195  OASL2P       111216278
#&gt;  7 rs2464195  XLOC_009911  105500240
#&gt;  8 rs2464195  HNF1A        6927     
#&gt;  9 rs2464195  C12orf43     64897    
#&gt; 10 rs2464195  RPL12P33     643550   
#&gt; # … with 81 more rows
#&gt; </div><div class='input'>
</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ramiro Magno, Ana-Teresa Maia, Centre for Biomedical Research, University of Algarve.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


