<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Subset an associations object — subset-associations • gwasrapidd</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



  <link href="../extra.css" rel="stylesheet">
  

<meta property="og:title" content="Subset an associations object — subset-associations" />
<meta property="og:description" content="You can subset associations by identifier or by position using the
`[` operator." />
<meta property="og:image" content="https://rmagno.eu/gwasrapidd/logo.svg" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">gwasrapidd</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.8</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/gwasrapidd.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/bmi_variants.html">Variants associated with Body Mass Index (BMI)</a>
    </li>
    <li>
      <a href="../articles/faq.html">Frequently asked questions</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/ramiromagno/gwasrapidd/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Subset an associations object</h1>
    <small class="dont-index">Source: <a href='https://github.com/ramiromagno/gwasrapidd/blob/master/R/generics.R'><code>R/generics.R</code></a></small>
    <div class="hidden name"><code>subset-associations.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>You can subset <a href='associations-class.html'>associations</a> by identifier or by position using the
<code><a href='https://rdrr.io/r/base/Extract.html'>`[`</a></code> operator.</p>
    </div>

    <pre class="usage"># S4 method for associations,missing,missing,missing
[(x, i, j, ..., drop = FALSE)

# S4 method for associations,numeric,missing,missing
[(x, i, j, ..., drop = FALSE)

# S4 method for associations,character,missing,missing
[(x, i, j, ..., drop = FALSE)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>x</th>
      <td><p>A <a href='associations-class.html'>associations</a> object.</p></td>
    </tr>
    <tr>
      <th>i</th>
      <td><p>Position of the identifier or the name of the identifier itself.</p></td>
    </tr>
    <tr>
      <th>j</th>
      <td><p>Not used.</p></td>
    </tr>
    <tr>
      <th>...</th>
      <td><p>Additional arguments not used here.</p></td>
    </tr>
    <tr>
      <th>drop</th>
      <td><p>Not used.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A <a href='associations-class.html'>associations</a> object.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='co'># Subset an associations object by identifier</span>
<span class='no'>associations_ex01</span>[<span class='st'>'22505'</span>]</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; # A tibble: 1 x 17
#&gt;   association_id  pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   &lt;chr&gt;            &lt;dbl&gt; &lt;chr&gt;                      &lt;int&gt;           &lt;int&gt;
#&gt; 1 22505          7.00e-6 &lt;NA&gt;                           7              -6
#&gt; # … with 12 more variables: multiple_snp_haplotype &lt;lgl&gt;,
#&gt; #   snp_interaction &lt;lgl&gt;, snp_type &lt;chr&gt;, standard_error &lt;dbl&gt;, range &lt;chr&gt;,
#&gt; #   or_per_copy_number &lt;dbl&gt;, beta_number &lt;dbl&gt;, beta_unit &lt;chr&gt;,
#&gt; #   beta_direction &lt;chr&gt;, beta_description &lt;chr&gt;, last_mapping_date &lt;dttm&gt;,
#&gt; #   last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "loci":
#&gt; # A tibble: 1 x 4
#&gt;   association_id locus_id haplotype_snp_count description   
#&gt;   &lt;chr&gt;             &lt;int&gt;               &lt;int&gt; &lt;chr&gt;         
#&gt; 1 22505                 1                  NA Single variant
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; # A tibble: 1 x 7
#&gt;   association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;      &lt;chr&gt;                &lt;dbl&gt; &lt;lgl&gt;      
#&gt; 1 22505                 1 chr22:458… T                       NA NA         
#&gt; # … with 1 more variable: limited_list &lt;lgl&gt;
#&gt; 
#&gt; Slot "genes":
#&gt; # A tibble: 2 x 3
#&gt;   association_id locus_id gene_name   
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;       
#&gt; 1 22505                 1 TBC1D22A    
#&gt; 2 22505                 1 LOC100128818
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 2 x 4
#&gt;   association_id locus_id gene_name    ensembl_id     
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;        &lt;chr&gt;          
#&gt; 1 22505                 1 TBC1D22A     ENSG00000054611
#&gt; 2 22505                 1 LOC100128818 &lt;NA&gt;           
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 2 x 4
#&gt;   association_id locus_id gene_name    entrez_id
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;        &lt;chr&gt;    
#&gt; 1 22505                 1 TBC1D22A     25771    
#&gt; 2 22505                 1 LOC100128818 &lt;NA&gt;     
#&gt; </div><div class='input'>
<span class='co'># Or by its position in table associations</span>
<span class='no'>associations_ex01</span>[<span class='fl'>2</span>]</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; # A tibble: 1 x 17
#&gt;   association_id  pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   &lt;chr&gt;            &lt;dbl&gt; &lt;chr&gt;                      &lt;int&gt;           &lt;int&gt;
#&gt; 1 22505          7.00e-6 &lt;NA&gt;                           7              -6
#&gt; # … with 12 more variables: multiple_snp_haplotype &lt;lgl&gt;,
#&gt; #   snp_interaction &lt;lgl&gt;, snp_type &lt;chr&gt;, standard_error &lt;dbl&gt;, range &lt;chr&gt;,
#&gt; #   or_per_copy_number &lt;dbl&gt;, beta_number &lt;dbl&gt;, beta_unit &lt;chr&gt;,
#&gt; #   beta_direction &lt;chr&gt;, beta_description &lt;chr&gt;, last_mapping_date &lt;dttm&gt;,
#&gt; #   last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "loci":
#&gt; # A tibble: 1 x 4
#&gt;   association_id locus_id haplotype_snp_count description   
#&gt;   &lt;chr&gt;             &lt;int&gt;               &lt;int&gt; &lt;chr&gt;         
#&gt; 1 22505                 1                  NA Single variant
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; # A tibble: 1 x 7
#&gt;   association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;      &lt;chr&gt;                &lt;dbl&gt; &lt;lgl&gt;      
#&gt; 1 22505                 1 chr22:458… T                       NA NA         
#&gt; # … with 1 more variable: limited_list &lt;lgl&gt;
#&gt; 
#&gt; Slot "genes":
#&gt; # A tibble: 2 x 3
#&gt;   association_id locus_id gene_name   
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;       
#&gt; 1 22505                 1 TBC1D22A    
#&gt; 2 22505                 1 LOC100128818
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 2 x 4
#&gt;   association_id locus_id gene_name    ensembl_id     
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;        &lt;chr&gt;          
#&gt; 1 22505                 1 TBC1D22A     ENSG00000054611
#&gt; 2 22505                 1 LOC100128818 &lt;NA&gt;           
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 2 x 4
#&gt;   association_id locus_id gene_name    entrez_id
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;        &lt;chr&gt;    
#&gt; 1 22505                 1 TBC1D22A     25771    
#&gt; 2 22505                 1 LOC100128818 &lt;NA&gt;     
#&gt; </div><div class='input'>
<span class='co'># Keep all associations except the second</span>
<span class='no'>associations_ex01</span>[-<span class='fl'>2</span>]</div><div class='output co'>#&gt; An object of class "associations"
#&gt; Slot "associations":
#&gt; # A tibble: 3 x 17
#&gt;   association_id   pvalue pvalue_descript… pvalue_mantissa pvalue_exponent
#&gt;   &lt;chr&gt;             &lt;dbl&gt; &lt;chr&gt;                      &lt;int&gt;           &lt;int&gt;
#&gt; 1 22509          3.00e- 9 &lt;NA&gt;                           3              -9
#&gt; 2 19537565       7.00e-31 &lt;NA&gt;                           7             -31
#&gt; 3 19537593       1.00e-11 &lt;NA&gt;                           1             -11
#&gt; # … with 12 more variables: multiple_snp_haplotype &lt;lgl&gt;,
#&gt; #   snp_interaction &lt;lgl&gt;, snp_type &lt;chr&gt;, standard_error &lt;dbl&gt;, range &lt;chr&gt;,
#&gt; #   or_per_copy_number &lt;dbl&gt;, beta_number &lt;dbl&gt;, beta_unit &lt;chr&gt;,
#&gt; #   beta_direction &lt;chr&gt;, beta_description &lt;chr&gt;, last_mapping_date &lt;dttm&gt;,
#&gt; #   last_update_date &lt;dttm&gt;
#&gt; 
#&gt; Slot "loci":
#&gt; # A tibble: 3 x 4
#&gt;   association_id locus_id haplotype_snp_count description   
#&gt;   &lt;chr&gt;             &lt;int&gt;               &lt;int&gt; &lt;chr&gt;         
#&gt; 1 22509                 1                  NA Single variant
#&gt; 2 19537565              1                  NA Single variant
#&gt; 3 19537593              1                  NA Single variant
#&gt; 
#&gt; Slot "risk_alleles":
#&gt; # A tibble: 3 x 7
#&gt;   association_id locus_id variant_id risk_allele risk_frequency genome_wide
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;      &lt;chr&gt;                &lt;dbl&gt; &lt;lgl&gt;      
#&gt; 1 22509                 1 rs7089814  C                       NA NA         
#&gt; 2 19537565              1 rs10110651 &lt;NA&gt;                    NA FALSE      
#&gt; 3 19537593              1 rs9397437  &lt;NA&gt;                    NA FALSE      
#&gt; # … with 1 more variable: limited_list &lt;lgl&gt;
#&gt; 
#&gt; Slot "genes":
#&gt; # A tibble: 7 x 3
#&gt;   association_id locus_id gene_name
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;    
#&gt; 1 22509                 1 ADO      
#&gt; 2 22509                 1 ZNF365   
#&gt; 3 22509                 1 ALDH7A1P4
#&gt; 4 19537565              1 KCNU1    
#&gt; 5 19537565              1 ZNF703   
#&gt; 6 19537593              1 CCDC170  
#&gt; 7 19537593              1 ESR1     
#&gt; 
#&gt; Slot "ensembl_ids":
#&gt; # A tibble: 7 x 4
#&gt;   association_id locus_id gene_name ensembl_id     
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;     &lt;chr&gt;          
#&gt; 1 22509                 1 ADO       ENSG00000181915
#&gt; 2 22509                 1 ZNF365    ENSG00000138311
#&gt; 3 22509                 1 ALDH7A1P4 ENSG00000234489
#&gt; 4 19537565              1 KCNU1     ENSG00000215262
#&gt; 5 19537565              1 ZNF703    ENSG00000183779
#&gt; 6 19537593              1 CCDC170   ENSG00000120262
#&gt; 7 19537593              1 ESR1      ENSG00000091831
#&gt; 
#&gt; Slot "entrez_ids":
#&gt; # A tibble: 7 x 4
#&gt;   association_id locus_id gene_name entrez_id
#&gt;   &lt;chr&gt;             &lt;int&gt; &lt;chr&gt;     &lt;chr&gt;    
#&gt; 1 22509                 1 ADO       84890    
#&gt; 2 22509                 1 ZNF365    22891    
#&gt; 3 22509                 1 ALDH7A1P4 544      
#&gt; 4 19537565              1 KCNU1     157855   
#&gt; 5 19537565              1 ZNF703    80139    
#&gt; 6 19537593              1 CCDC170   80129    
#&gt; 7 19537593              1 ESR1      2099     
#&gt; </div><div class='input'>
</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Ramiro Magno, Ana-Teresa Maia, Centre for Biomedical Research, University of Algarve.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


